CHANGELOG.md

```diff
@@ -6,6 +6,23 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
 ## [Unreleased]
 
+### Added
+
+- Uncarved Wood Variants (Polished uncarved Wood, Carved Heavy Planks, Carved Knotted Wood, Carved Planks).
+- Combinations for Uncarved Wood Variants.
```

Comment:
"Combinations" is too vague, and should be "Combination recipes" instead. However because the items that the recipes were made for is added in the same update, I suggest that you remove the recipes from the changelog instead.
```suggestion
```